docs/azure/aksextensions.md

```diff
@@ -0,0 +1,127 @@
+---
+Order: 7
+Area: azure
+TOCTitle: AKS Extensions
+ContentId: xxx
+PageTitle: Working with AKS tools and diagnostics in Visual Studio Code
+DateApproved: 5/03/2022
+MetaDescription: Working with AKS tools and diagnostics in Visual Studio Code
+---
+# Working with AKS tools and diagnostics in VS Code
+
+This document will walk you through the process of using AKS features, AKS Diagnostics healthchecks and one-click AKS specific tools which can be used for AKS clusters with Visual Studio Code. [Azure Kubernetes Services](https://azure.microsoft.com/en-in/services/kubernetes-service/#overview) is a fully managed [Kubernetes](https://azure.microsoft.com/en-in/topic/what-is-kubernetes/#overview) service. Azure Kubernetes Service (AKS) offers serverless Kubernetes, an integrated continuous integration and continuous delivery (CI/CD) experience and enterprise-grade security and governance. is an open-source system for automating deployment, scaling, and management of containerized applications. We will show you how to keep diagnostics health-checks on your AKS cluster, run tools like aks-periscope, or deploy Azure service operator or generate infrastructure startup-AKS workflows.
+
+## Before you begin
+
+The [AKS Visual Code Extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) has parent dependency on [Kubernetes Visual Code extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools). You will need to download the [kubernetes VS Code extension](https://code.visualstudio.com/docs/azure/kubernetes).
+
+## Install the Azure Kubernetes Services extension
+
+For a fully integrated Azure Kubernetes diagnostics experience, you can install the [Azure Kubernetes Services Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) extension.
+
+With the extension, you can quickly develop, deploy Azure Kubernetes Diagnostics health-checks, run diagnosics tools to collect cluster logs like aks-periscope, create GitHub Starter workflows or deploy services like azure service operator in jsut one-click.
```

Comment:
Spelling: 'jsut'... Also, we maybe don't need to repeat the individual things you can do here because they're detailed above and below. I'm wondering about something like:

> With the extension you can easily view and manage your AKS clusters from your Visual Studio Code environment.